src/utils/HTMLUtils/index.js

```diff
@@ -39,23 +51,28 @@ const mapElementToObject = elem => {
   }
 };
 
-const parseContent = html => contentType => {
-  const content = cheerio(html)
+const parseContent = html => {
+  if (!html) {
+    return;
+  }
+
+  const text = html.text ? html.text : html;
+  const content = cheerio(text)
     .filter((i, elem) => getInnerHTML(elem) !== "")
     .map((i, elem) => mapElementToObject(elem))
     .toArray();
 
-  if (content.length === 0) {
+  if (!content.length) {
     return;
   }
```

Comment:
Should this throw an error instead of return undefined? Just thinking about when it comes to debugging later on.